qiita_db/support_files/patches/71.sql

```diff
@@ -4,15 +4,18 @@
 -- 4/4 This is the continuation of the patching that started in 68.sql, let's
 -- remove all the temp (_bk) tables we created
 
--- Dropping all the _bk tables
+-- On January 17, 2019 while releasing the release candidate; we realized that
+-- we were hitting the max memory allocated for the machine so we needed to
+-- split this in to 2
+-- Dropping all the _bk tables 1/2
 DO $do$
 DECLARE
     dyn_table varchar;
 BEGIN
   FOR dyn_table IN
     SELECT DISTINCT table_name
     FROM information_schema.columns
-    WHERE table_name LIKE '%_bk'
+    WHERE table_name LIKE '%_bk' LIMIT 2100
```

Comment:
Will the hardcoded 2100 limit need to be updated in the future, as the number of tables grow? We might need a comment here whether this will be an issue in the future.

Reply:
patches are only ran once, so it should not be a problem in the future ...